commander/src/commands/transaction/sign.ts

```diff
@@ -88,7 +128,32 @@ export default class SignCommand extends BaseCommand {
 			...transactionObject,
 			networkIdentifier,
 		});
-		txInstance.sign(passphrase);
+
+		// tslint:disable-next-line: no-object-literal-type-assertion
+		const keys = {
+			mandatoryKeys,
+			optionalKeys,
+			numberOfSignatures,
+		} as {
```

Comment:
no need to type cast here.
However `numberOfSignatures` can be undefined. 
but in the `signMultiSignatureTransaction` it seems like it's not used  although it is in argument.
maybe we can remove from the flag and the function? @pablitovicente

Reply:
I think we can remove.. its not that required. let me know whats your opinion pablo.

Reply:
it's there because the interface was designed like that but it can be removed as it only make sense during verification which in that stage the transaction will need the account to verify what was the configuration at registration time.

Reply:
ok then I will remove it 👍